contents/teams/growth/objectives.mdx

```diff
@@ -1,27 +1,20 @@
-  - Metrics: We have the feature, and people use it.
-  - Side hustle, if time permits: Figure out how to do experiments with pricing.
-- **Objective 3**: Dive into the data to understand our billing metrics and customers better (Zach)
-  - Reasoning: We want to know:
-      - How the startup program is performing and how we should change things there
-      - How well we cross-sell and how that can be improved
-      - Why CC input rate is going down, even though signups are going up.
-      - If we can trust the revenue number every single day
-  - Metrics: We have access to more stripe data in billing (i.e. stripe customer metadata), and we can answer the questions above.
-- **Objective 4**: Keep up with ongoing needs from our team:
-  - Hire 2 people onto the team - one billing focused, one auth/permissions or growth focused (Raquel)
-  - Launch pricing for Data warehouse (likely soon) (Raquel)
-  - Launch pricing for CDP (if it's ready)
+- **Objective 1**: Get to a 4/10 in Enterprise platform support (Zach)
+  - Release Enterprise addon and iterate with sales to find a pricing model that makes sense
+  - Build RBAC
+  - Reasoning: As a company we want to have a 4/10 support for enterprise - and our platform needs to support that.
+  - Metrics: RBAC is at a place where enterprises who require it can actually use it.
+- **Objective 2**: Make revenue & forecasting more reliable (Patricio)
+  - Rejig how we do sync_invoices so we don’t see big drops - respond to webhooks instead of large syncs
+  - Evaluate temporal vs celery for usage reports and billing tasks so we can adhere to rate limits better
+  - Side hustle, if time permits: Rejig how anonymous / identified events are billed, as the current implementation is confusing.
+- **Objective 3**: Activation: reverse trials, analytics onboarding flow, understand CC activation (Raquel)
+  - Dig into the data, watch replays to figure out why CC activation rate has been falling
+  - Add reverse trial to onboarding
+  - Add reverse trial to billing page for teams
+  - Reasoning: Our CC activation rate has been falling and we aren't really sure why. We should figure it out and try a few things to improve it. I also want to continue my focus on analytics activation that I finally got to start last quarter that I didn't get to finish!
+  - Metrics: Improved CC activaiton rate. Increase in people who sign up for Teams. Increase in people who successfully activate into product analytics and replay.
 
-If we have time:
-  - Build a better billing page that gives breakdowns by project, is easier to mentally parse, is more beautiful, etc. 
-  - Fix all the bugs and unknowns with auth (eg why xmlsec gives us so many problems)
-  - Move CC entry into the app
+Stretch goals:
```

Comment:
Another one we can add: move the annual credits flow out of Zapier.

Reply:
I knew I was forgetting something!